disasm-test/known_bugs/llvm-pretty_issue13.txt

```diff
@@ -0,0 +1,12 @@
+##> rootMatchName: callbr.c
+##> summary: DICompileUnit rangesBaseAddress only added in LLVM 11
+##> llvmver: 10
```

Comment:
The format uses here doesn't quite appear to agree with what is documented in the known bugs `README`:

https://github.com/GaloisInc/llvm-pretty-bc-parser/blob/ad6fd5c04576c914b3dd4fbda9fff0b2f68c25d1/disasm-test/known_bugs/README.md?plain=1#L32-L34

Should the `README` be updated to say `N` instead of `llvmN`?

Reply:
Good catch, thanks. I fixed the handling of `llvmver` marker lines in this PR, but forgot to check this README.